payloads/github_api/repos_dlang_dmd_pulls?state=open.json

```diff
@@ -0,0 +1,9128 @@
+[
```

Comment:
Yeah probably, but it's compressed in the git history anyways and locally it doesn't matter?
The entire payloads folder has less than 1M atm.